Softeq.XToolkit.Connectivity/Softeq.XToolkit.Connectivity.csproj

```diff
@@ -1,9 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>

```

Comment:
Please set lang version and `<Nullable>` in the shared `<PropertyGroup>`, instead of build configurations (VS default behavior)